.github/workflows/SPM Testing.yml

```diff
@@ -1,17 +1,21 @@
 # This is the First Workflow for Testing with Github Actions
 
 name: SPM Check
+
 on:
-  workflow_dispatch:
-env:
-  DEVELOPER_DIR: /Applications/Xcode_12.4.app/Contents/Developer
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
 jobs:
   spm-check:
-    runs-on: macos-10.15
+    runs-on: macos-latest
     steps:
-    - uses: actions/checkout@v2
-    - name: Setup Package File
-      run: mv .Package.Test.swift Package.swift
```

Comment:
Deleted the `.Package.Test.swift` file so we always use `Package.swift`
ref: https://github.com/Swinject/Swinject/pull/517